index.html

```diff
@@ -38,197 +51,234 @@ <h1 class="hero-title">
-                <h3 class="education-title">Frameworks</h3>
-                <ul class="tags">
-                    <li class="primary-font-style">Bootstrap</li>
-                    <li class="primary-font-style">Ruby on rails</li>
-                    <li class="primary-font-style">RSpec</li>
-                    <li class="primary-font-style">Capybara</li>
-                    <li class="primary-font-style">Selenium</li>
-                </ul> 
-            </div>
-        </div>
-        </section>
 
-        <section class="education-card-3">
-        <div class="education-card">
-            <img src="Images/icon-skills.png" alt="Icon Skills">
-            <div class="education-card-footer">
-                <h3 class="education-title">Skills</h3>
-                <ul class="tags">
-                    <li class="primary-font-style">Codekit</li>
-                    <li class="primary-font-style">GitHub</li>
-                    <li class="primary-font-style">Codepen</li>
-                    <li class="primary-font-style">Gitlab</li>
-                    <li class="primary-font-style">Terminal</li>
-                </ul> 
-            </div>
+    <!-- Footer and contact form -->
+
+
+    <footer class="contact-form-section">
+        <div id="contact-form-detail">
+        <div class="contact-form-description">
+            <h2 id="contact-form-desc">
+                I'm always interested in hearing
+                about new projects, so if you'd 
+                like to chat please get in touch.
+            </h2>
         </div>
-        </section>
-    </section>
+        <div class="form">
+            <form action="https://formspree.io/f/mpznrkvl" method="post">
+                <input type="text" id="fullname" inputname="fullname" maxlength="30" placeholder="Full Name" required><br>
+                <input type="email" id="email" inputname="" placeholder="Email Address" required>
+                <textarea name="message" id="textarea" placeholder="Enter Text Here" maxlength="500" required></textarea>
+                <input type="submit" value="Get in touch" id="get-in-btn" class="standard-button">
+            </form>
+        </div>
+    </div>
+        <div id="social-media-links-2">
+            <ul id="media-links">
+                <li><a href="https://github.com/187jjay187"><img src="Images/github.svg" alt="github"></a></li>
+                <li><a href="https://linkedin.com"><img src="Images/Linkedin icon.svg" alt="Linkedin logo"></a></li>
+                <li><a href="https://angel.co"><img src="Images/angellist.svg" alt="angellist"></a></li>
+                <li><a href="https://twitter.com"><img src="Images/twitter.svg" alt="twitter"></a></li>
+                <li><a href="https://medium.com"><img src="Images/medium.svg" alt="medium"></a></li>
+            </ul>
+            <p id="footer-email">187jjay187@gmail.com</p>
+        </div>
+    </footer>
+    <!-- footer section end here -->
```

Comment:
- [ ] Kndly add a background image to this section to match the figma design. You are expected to stick to the figma template as much as possible.
- [ ] Kindly edit the color of the borders of the name and email input fields to match design template.
### Your design
![3](https://user-images.githubusercontent.com/94551484/194693353-cb74eab5-f0cb-4c3c-a69e-2be01296d448.png)

### Figma template
![f3](https://user-images.githubusercontent.com/94551484/194693363-e8c7aebb-1313-4c2c-a96e-47199ef3b37f.png)